### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by AnkitaSikdar005**
## What does this PR do?
Add resource(s)

## For resources
Added Figma Course in Hindi

### Why is this valuable (or not)?
Yes, it is valuable.
### How do we know it's really free?
It is available on YouTube.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by AnkitaSikdar005** [comment]
@eshellman Please merge my pull request.

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -274,6 +274,7 @@
 ### Figma
 
 * [Figma Tutorial](https://www.youtube.com/playlist?list=PLuRPummNMvINdAbI_WT7R5vdjcyRPeRiq) - Pelfizz Studio
+* [Figma Tutorial For Beginners in Hindi](https://www.youtube.com/watch?v=UjPpvf4TNLI) - LearnGrow
```
Comment (eshellman) [inline courses/free-courses-hi.md:277]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

**Comment by AnkitaSikdar005** [comment]
@eshellman Made the changes as told by you.

**Comment by eshellman** [comment]
why not the Urdu list?

**Comment by eshellman** [comment]
added in #10980

**Pull request closed without merging**